src/models/data.cpp

```diff
@@ -133,6 +133,158 @@ ItemCost buildCallerCalleeResult(const BottomUp& data, const Costs& bottomUpCost
+            || str[pos - 6] == QLatin1Char('<')
+            || str[pos - 6] == QLatin1Char(' ')
+            || str[pos - 6] == QLatin1Char('(')) {
+            break;
+        }
+    } while (true);
+
+    auto result = str.left(pos).toString();
+    auto symbol = str.mid(pos);
+
+    int end;
+    if (startsWith(symbol, QLatin1String("__cxx11::"), &end)
+        || startsWith(symbol, QLatin1String("__1::"), &end)) {
+        // Strip libstdc++/libc++ internal namespace
+        symbol = symbol.mid(end);
+    }
+
+    if (startsWith(symbol, QLatin1String("basic_string<"), &end)) {
+        const int comma = findSameDepth(symbol, end, QLatin1Char(','));
+        if (comma != -1) {
+            const auto type = symbol.mid(end, comma - end);
+            if (type == QLatin1String("char")) {
+                result += QLatin1String("string");
+            }
+            else if (type == QLatin1String("wchar_t")) {
+                result += QLatin1String("wstring");
+            }
+            else {
+                result += symbol.left(end);
+                result += type;
+                result += QLatin1Char('>');
+            }
+            end = findSameDepth(symbol, 0, QLatin1Char('>'), true);
+            symbol = symbol.mid(end);
+
+            if (startsWith(symbol, QLatin1String("::basic_string("), &end)
+                || startsWith(symbol, QLatin1String("::~basic_string("), &end)) {
+                result += QLatin1String("::");
+                if (symbol[2] == QLatin1Char('~')) {
+                    result += QLatin1Char('~');
+                }
+                if (type == QLatin1String("wchar_t")) {
+                    result += QLatin1Char('w');
+                }
+                else if (type != QLatin1String("char")) {
+                    result += QLatin1String("basic_");
+                }
+                result += QLatin1String("string(");
+                symbol = symbol.mid(end);
+            }
+        }
+    }
+    else if (startsWith(symbol, QLatin1String("vector<"), &end)
+             || startsWith(symbol, QLatin1String("set<"), &end)
+             || startsWith(symbol, QLatin1String("deque<"), &end)
+             || startsWith(symbol, QLatin1String("list<"), &end)
+             || startsWith(symbol, QLatin1String("forward_list<"), &end)
+             || startsWith(symbol, QLatin1String("multiset<"), &end)
+             || startsWith(symbol, QLatin1String("unordered_set<"), &end)
+             || startsWith(symbol, QLatin1String("unordered_multiset<"), &end)) {
+        const int comma = findSameDepth(symbol, end, QLatin1Char(','));
+        if (comma != -1) {
+            result += symbol.left(end);
+            result += prettifySymbol(symbol.mid(end, comma - end));
+            result += QLatin1Char('>');
+
+            end = findSameDepth(symbol, 0, QLatin1Char('>'), true);
+            symbol = symbol.mid(end);
+        }
+    }
+    else if (startsWith(symbol, QLatin1String("map<"), &end)
```

Comment:
ah, since this is the same helper function here, let the lambda take the initializer_list of type names as an argument and then use it here again